apps/web-giddh/src/app/shared/sales-person/sales-person.component.ts

```
@@ -152,7 +168,20 @@ export class SalesPersonComponent implements OnInit, AfterViewInit, OnDestroy {
                 break;
             case SalesPersonActionEnum.DELETE:
-                this.componentStore.deleteSalesPerson(element?.uniqueName);
+                const dialogRef = this.dialog.open(NewConfirmationModalComponent, {
+                    panelClass: ['mat-dialog-sm'],
+                    data: {
+                        configuration: this.generalService.deleteConfiguration(
+                            this.commonLocaleData?.app_permanently_delete_message,
+                            this.commonLocaleData
+                        )
+                    }
+                });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)